### DISCUSSION TITLE
Question returning nothing in Windows

### DISCUSSION THREAD

**Discussion by josuigoa** [ncruces/zenity#32] [Q&A] [answered]
Hi!

firts of all, thank you for creating this library, it is very useful!

I'm running the latest windows binary (0.8.9) from command line like this:

`zenity.exe --question --text "are you sure?" --title "test"`

And it doesn't matter what button I'm pressing, it returns empty string (or nil, I don't know go :-) )

thanks in advance

**Comment by ncruces** [comment, marked as answer]
Hi!

The `zenity --question` command returns its result in the exit code of the process.

Depending on your shell (`cmd`, `powershell`, Git `bash`) this should work:
```zenity --question --text "are you sure?" --title "test" && echo yes || echo no```

**Comment by josuigoa** [comment]
oohh, I was expecting the response in the standard output. The exit code option is also logical, I didn't thought about it!

thanks!
